solutions/src/main/java/ru/sberbank/school/task02/ExternalQuotesProvider.java

```diff
@@ -0,0 +1,21 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.util.Volume;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExternalQuotesProvider implements ExternalQuotesService {
+    @Override
+    public List<Quote> getQuotes(Symbol symbol) {
+        List<Quote> quotes = new ArrayList<>();
+        for(int i = 9; i > 0; i--) {
```

Comment:
WhitespaceAround: &#39;for&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a446468950>*